backend/bitboxbase/bitboxbase.go

```diff
@@ -254,20 +286,205 @@ func (base *BitBoxBase) SyncWithOption(option SyncOption) (bool, error) {
 		base.changeStatus(bitboxbasestatus.StatusInitialized)
 		return true, nil
 	case SyncOptionReindex:
-		base.log.Println("bitboxbase is making a ReindexResyncOption call")
-		replySuccess, err := base.rpcClient.ResyncBitcoin(rpcmessages.Reindex)
+		base.log.Println("bitboxbase is making a ReindexBitcoin call")
+		replySuccess, err := base.rpcClient.ReindexBitcoin()
 		base.changeStatus(bitboxbasestatus.StatusInitialized)
 		return replySuccess.Success, err
```

Comment:
This function should be split up into two to be consistent with the other ones in this file.

Reply:
Yes, let's split it up before merging